resources/public/api/conf/4.0/foreign_property_retrieve.yaml

```
@@ -12,22 +12,50 @@ get:
+    | FOREIGN_PROPERTY_NON_FHL_PROFIT               | Simulates success response with non FHL foreign property and profit.                |
+    | DYNAMIC_FOREIGN_PROPERTY_NON_FHL_PROFIT       | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_PROPERTY_NON_FHL_LOSS                 | Simulates success response with non FHL foreign property and loss.                  |
+    | DYNAMIC_FOREIGN_PROPERTY_NON_FHL_LOSS         | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_PROPERTY_FHL_CONSOLIDATED             | Simulates success response with FHL adjusted summary and consolidated expenses.     |
+    | DYNAMIC_FOREIGN_PROPERTY_FHL_CONSOLIDATED	    | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_PROPERTY_NON_FHL_CONSOLIDATED         | Simulates success response with non FHL adjusted summary and consolidated expenses. |
+    | DYNAMIC_FOREIGN_PROPERTY_NON_FHL_CONSOLIDATED	| See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_FHL_PROPERTY_ALLOWANCE                | Simulates success response with FHL adjusted summary and propertyAllowance.         |
+    | DYNAMIC_FOREIGN_FHL_PROPERTY_ALLOWANCE        | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_NON_FHL_PROPERTY_ALLOWANCE            | Simulates success response with non FHL adjusted summary and propertyAllowance.     |
+    | DYNAMIC_FOREIGN_NON_FHL_PROPERTY_ALLOWANCE    | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_PROPERTY_STATUS_INVALID               | Simulates success response with summary status invalid.                             |
+    | DYNAMIC_FOREIGN_PROPERTY_STATUS_INVALID	    | See below for details on DYNAMIC scenarios.                                         |
+    | FOREIGN_PROPERTY_STATUS_SUPERSEDED            | Simulates success response with summary status superseded.                          |
+    | DYNAMIC_FOREIGN_PROPERTY_STATUS_SUPERSEDED    | See below for details on DYNAMIC scenarios.                                         |
+    | NOT_FOREIGN_PROPERTY                          | Simulates the scenario where the calculationId is not a foreign property.           |
+    | TAX_YEAR_NOT_SUPPORTED                        | Simulates the scenario where the tax year is not supported.                         |
+    | NOT_FOUND                                     | Simulates the scenario where no matching resource is found.                         |
+    
+    In each case where a scenario is prefixed by DYNAMIC_, the following response values will change to correspond to the values submitted in the request:
+      * metadata.
+        * calculationId
+        * taxYear
+        * nino
+        * requestedDateTime
+        * adjustedDateTime
+      * inputs.
```

I think you should remove the dots. They're not in the tech spec and I don't think they convey the concatenation of two fields.
```suggestion
      * metadata
        * calculationId
        * taxYear
        * nino
        * requestedDateTime
        * adjustedDateTime
      * inputs
```